zanata-war/src/main/java/org/zanata/action/ActivateAction.java

```diff
@@ -40,34 +40,37 @@
 import org.zanata.seam.security.IdentityManager;
 import org.zanata.ui.faces.FacesMessages;
 
+import lombok.Getter;
+import lombok.Setter;
+
 @Name("activate")
 @Scope(ScopeType.CONVERSATION)
 public class ActivateAction implements Serializable {
 
     private static final long serialVersionUID = -8079131168179421345L;
 
+    private static int LINK_ACTIVE_DAYS = 1;
+
     @In
     private AccountActivationKeyDAO accountActivationKeyDAO;
 
     @In
     private IdentityManager identityManager;
 
+    @In
+    private Redirect redirect;
```

Comment:
Could we use UserRedirect instead of Seam's Redirect? This will make it hard to migrate to CDI.